From d5ad538adb9fcbad8a3b4e46f266b668301cb1c4 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Wed, 2 Mar 2022 05:36:48 +0000
Subject: [PATCH] configure.ac: Don't use AC_CHECK_FILE

AC_CHECK_FILE is not suitable for cross-compile, so
remove it to fix configure error:
cannot check for file existence when cross compiling

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 configure.ac | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 775c795..bd99188 100644
--- a/configure.ac
+++ b/configure.ac
@@ -116,9 +116,6 @@ AC_TYPE_SIZE_T
 AC_FUNC_MALLOC
 AC_CHECK_FUNCS([memset strdup strstr])
 
-# Bash@Ubuntu@Windows
-AC_CHECK_FILE(/tmp/hstr-ms-wsl,AC_DEFINE(__MS_WSL__), [])
-
 # Bash CLI autocomplete
 AC_ARG_WITH([bash-completion-dir],
     AS_HELP_STRING([--with-bash-completion-dir[=PATH]],
-- 
2.31.1

